docker-run-sync-rapid-pro-to-engagement-db.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -e
+
+PROJECT_NAME="$(<configurations/docker_image_project_name.txt)"
+IMAGE_NAME=$PROJECT_NAME-rapid-pro-to-engagement-db
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --incremental-cache-path)
+            INCREMENTAL_MODE=true
+            INCREMENTAL_CACHE_PATH="$2"
+            shift 2;;
+        --local-archive)
+            USE_ARCHIVE=true
+            LOCAL_ARCHIVE="$2"
+            shift 2;;
+        --)
+            shift
+            break;;
+        *)
+            break;;
+    esac
+done
+
+# Check that the correct number of arguments were provided.
+if [[ $# -ne 3 ]]; then
+    echo "Usage: ./docker-run-sync-rapid-pro-to-engagement-db.sh
+    [--incremental-cache-path <incremental-cache-path>] [--local-archive <local_archive>]
+    <user> <google-cloud-credentials-file-path> <configuration-module>"
+    exit
+fi
+
+# Assign the program arguments to bash variables.
+USER=$1
+INPUT_GOOGLE_CLOUD_CREDENTIALS=$2
```

Comment:
```suggestion
GOOGLE_CLOUD_CREDENTIALS_PATH=$2
```
(I know you copied this from before, but I think this is better)